.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -45,7 +45,6 @@ body:
         # - "avm/ptn/avd-lza/management-plane"
         # - "avm/ptn/avd-lza/networking"
         # - "avm/ptn/avd-lza/session-hosts"
-        - "avm/ptn/policy-insights/remediation"
```

Comment:
Is this removal intended?

Reply:
@eriqua I fixed this. my fork was messed up and had some weird commits in it that were getting dragged along into all my other branches. This should be a clean commit now with just the changes related to the fix.